src/main/java/org/mskcc/cmo/messaging/impl/MessageConsumerImpl.java

```diff
@@ -1,17 +1,17 @@
 package org.mskcc.cmo.messaging.impl;
 
 import org.mskcc.cmo.messaging.MessageConsumer;
-
-import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.commons.logging.Log;
+
 
 public class MessageConsumerImpl implements MessageConsumer {
 
-    private final Log LOG = LogFactory.getLog(MessageConsumerImpl.class);
+    private final Log logger = LogFactory.getLog(MessageConsumerImpl.class);
```

Comment:
I'm not sure the point of this class.  Don't we want to "force" clients to implement this?